packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/plugins/core/common.tsx

```diff
@@ -59,6 +59,7 @@ export const unsupportedItemsInTable = new Set([
   'Unordered List',
```

Comment:
I think we should change this at a later stage, I feel its going to be forgotten and someones going to change the label/tooltip of one of the buttons and this will break.